### PR TITLE
Upgrade jQuery, momentjs & leaflet dependencies

### PR DESCRIPTION
**Pull request by joehybird**

### CODE REVIEW

File: creme/creme_core/views/testjs.py [hunk 1]
```diff
@@ -102,11 +102,7 @@ def __init__(self, name, user, image_url):
         self.duration = mark_safe(print_duration(
             self, f'{randint(0, 23)}:{randint(0, 59)}:{randint(0, 59)}', user, None
         ))
-        self.foreignkey = (
-            None
-            if property is None else
-            mark_safe(print_foreignkey_html(self, property, user, None))
-        )
+        self.foreignkey = mark_safe(print_foreignkey_html(self, property, user, None))
```
Comment (genglert) [inline creme/creme_core/views/testjs.py:105]: It's the built-in "property" ? :thinking:

**Review by genglert — CHANGES_REQUESTED** [review]
Is the commit for jQuery will be backported to 2.4?

**Comment by joehybird** [comment]
> Is the commit for jQuery will be backported to 2.4?

Yes, I didn't want to cause any side effect for the 2.4.0 release but it can be backported without problem.

File: creme/creme_core/views/testjs.py (hunk 1 above)
Comment (joehybird) [reply creme/creme_core/views/testjs.py:105]: :facepalm: I DO need holidays

**Review by genglert — APPROVED** [review]

**Pull request merged**